polaris.dme

```diff
@@ -1,3277 +1,3276 @@
-#include "code\controllers\subsystems\radiation.dm"
-#include "code\controllers\subsystems\shuttles.dm"
-#include "code\controllers\subsystems\skybox.dm"
-#include "code\controllers\subsystems\sqlite.dm"
-#include "code\controllers\subsystems\sun.dm"
-#include "code\controllers\subsystems\supply.dm"
-#include "code\controllers\subsystems\tgui.dm"
-#include "code\controllers\subsystems\ticker.dm"
-#include "code\controllers\subsystems\time_track.dm"
-#include "code\controllers\subsystems\timer.dm"
-#include "code\controllers\subsystems\vote.dm"
-#include "code\controllers\subsystems\webhooks.dm"
-#include "code\controllers\subsystems\xenoarch.dm"
-#include "code\controllers\subsystems\processing\fastprocess.dm"
-#include "code\controllers\subsystems\processing\obj.dm"
-#include "code\controllers\subsystems\processing\processing.dm"
-#include "code\controllers\subsystems\processing\projectiles.dm"
-#include "code\controllers\subsystems\processing\turfs.dm"
-#include "code\datums\ai_law_sets.dm"
-#include "code\datums\ai_laws.dm"
-#include "code\datums\beam.dm"
-#include "code\datums\browser.dm"
-#include "code\datums\callback.dm"
-#include "code\datums\category.dm"
-#include "code\datums\chat_message.dm"
-#include "code\datums\computerfiles.dm"
-#include "code\datums\datacore.dm"
-#include "code\datums\datum.dm"
-#include "code\datums\datumvars.dm"
-#include "code\datums\EPv2.dm"
-#include "code\datums\ghost_query.dm"
-#include "code\datums\hierarchy.dm"
-#include "code\datums\mind.dm"
-#include "code\datums\mixed.dm"
-#include "code\datums\modules.dm"
-#include "code\datums\mutable_appearance.dm"
-#include "code\datums\orbit.dm"
-#include "code\datums\organs.dm"
-#include "code\datums\position_point_vector.dm"
-#include "code\datums\progressbar.dm"
-#include "code\datums\riding.dm"
-#include "code\datums\soul_link.dm"
-#include "code\datums\sun.dm"
-#include "code\datums\weakref.dm"
-#include "code\datums\autolathe\arms.dm"
-#include "code\datums\autolathe\autolathe.dm"
-#include "code\datums\autolathe\devices.dm"
-#include "code\datums\autolathe\engineering.dm"
-#include "code\datums\autolathe\general.dm"
-#include "code\datums\autolathe\materials.dm"
-#include "code\datums\autolathe\medical.dm"
-#include "code\datums\autolathe\tools.dm"
-#include "code\datums\components\_component.dm"
-#include "code\datums\components\crafting\crafting.dm"
-#include "code\datums\components\crafting\crafting_external.dm"
-#include "code\datums\components\crafting\recipes.dm"
-#include "code\datums\components\crafting\tool_quality.dm"
-#include "code\datums\elements\_element.dm"
-#include "code\datums\game_masters\_common.dm"
-#include "code\datums\game_masters\default.dm"
-#include "code\datums\game_masters\other_game_masters.dm"
-#include "code\datums\helper_datums\construction_datum.dm"
-#include "code\datums\helper_datums\events.dm"
-#include "code\datums\helper_datums\getrev.dm"
-#include "code\datums\helper_datums\global_iterator.dm"

```

Comment:
Oh that's where the diff got so big I was worried for a moment

Reply:
WHOOPS goddammit

Reply:
The removal of events.dm and global_iterator.dm was intentional, the other two... Weren't, but it compiles still somehow. I'll readd them assuming they actually do something.

Reply:
Rather, you changed the lineendings in the .dme so it marked the _whole_ file as deleted/added